source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+`cert-manager <https://cert-manager.io/>`__ works as a controller for managing certificates within Kubernetes clusters.
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can renew certificates prior to expiration.
+
+The MinIO Operator supports using cert-manager for certificates as an alternative to the MinIO Operator managing certificates for itself and its tenants.
+
+On a Kubernetes cluster, cert-manager requires a global level ``ClusterIssuer`` to generate intermediary ``Issuers`` and certificates.
+At the namespace level, cert-manager issues certificates derived from an ``Issuer``.
+A ``ClusterIssuer`` can issue certificates for multiple namespaces.
+An ``Issuer`` only issues certificates for its own namespace.
+
+To learn more about cert-manager Issuers, refer to the `Issuer documentation <https://cert-manager.io/docs/concepts/issuer/>`__.
+
+Below is a logical view of a Kubernetes cluster with four namespaces:
+
+- ``minio-operator``
+- ``tenant-1``
+- ``tenant-2``
+- ``other-namespace``
```

Comment:
We never reference this again, i'm not sure it does much for us. We can limit scope to just how cert manager works w/ minio resources.

Reply:
I removed this entirely in an IA rework of the page into multiple pages.